src/home/rendering/render-github-issues.ts

```diff
@@ -75,11 +81,10 @@ function setUpIssueElement(
   const image = `<img />`;
 
   issueElement.innerHTML = `
-      <div class="info"><div class="title"><h3>${
-        task.preview.title
-      }</h3></div><div class="partner"><p class="organization-name">${organizationName}</p><p class="repository-name">${repositoryName}</p></div></div><div class="labels">${labels.join(
-        ""
-      )}${image}</div>`;
+      <div class="info"><div class="title"><h3>${task.preview.title
+    }</h3></div><div class="partner"><p class="organization-name">${organizationName}</p><p class="repository-name">${repositoryName}</p></div></div><div class="labels">${labels.join(
+      ""
+    )}${image}</div>`;
```

Comment:
What is this?

Reply:
format-on-save I believe, no actual changes were made here